modules/test/playwright/fixtures/headless.fixture.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {test} from '@playwright/test';
+
+import {OpenAPI} from '../headless';
+import {liferayConfig} from '../liferay.config';
+
+const headlessTest = test.extend<{_headless: void}>({
+	_headless: [
+		async ({}, use) => {
+			OpenAPI.USERNAME = liferayConfig.user.login;
```

Comment:
I would initialize this in the `headless/index.ts` module explicitly.

This is because:

1. The API helpers are static classes that can be used from anywhere so they shouldn't depend on a fixture to be initialized.  
2. The fixture is not really providing or enforcing anything so someone can use the API helpers and they can work by coincidence (because the fixture was invoked from other test) or fail. This can lead to flaky tests.

In the future, you can provide a fixture that lets the caller specify the user and password, for example, and make the fixture inject an object to call the helpers (instead of using a static class).

Or, if you prefer, you can do that now and forbid accessing API helpers from any point of code not declaring the fixture.

That seems much cleaner to me...